pyvista/core/dataset.py

```diff
@@ -1232,6 +1232,31 @@ def length(self: Self) -> float:
         """
         return self.GetLength()
 
+    @property
+    def size(self: Self) -> tuple[float, float, float]:
```

Comment:
This term is heavily used, e.g. the `Label.size` property deprecated in this PR, but also many others:
https://dev.pyvista.org/search.html?q=size

This makes the term ambiguous, maybe call it `bounds_size` instead? Or maybe `physical_size`... or something else? With this change we could also remove the deprecation of `Label.size`.

```suggestion
    def bounds_size(self: Self) -> tuple[float, float, float]:
```

Reply:
Since the values are obtained from bounds internally, the name `bounds_size` feels more natural.